### PR TITLE
Blackout Footer

### LINKED ISSUE #39: Blackout footer
Footer as in the design spec

### PR DESCRIPTION
**Pull request by roball24**
Fixes #39 .

### PR Description
Just needed to start over on this

### What needs to be reviewed?
footer

### CODE REVIEW

File: app/pages/blackout/footer.jsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { Container } from '../../components';
+import { devices } from '../../styles';
+
+const HeaderLogoImage = require('../../../static/icons/blackout-logo.png');
+const InstagramImg = require('../../../static/icons/Instagram.png');
+const FacebookImg = require('../../../static/icons/Facebook.png');
```
Comment (connorkrupp) [inline app/pages/blackout/footer.jsx]: Lowercase file names pl0x

File: app/pages/blackout/index.jsx [hunk 2]
```diff
@@ -25,11 +26,7 @@ const StyledFaq = styled(Faq)``;
 class BlackoutPage extends React.Component {
     render() {
         return (
-            <Page>
-                <Header />
-                <StyledHero />
-                <StyledFaq />
-            </Page>
+            <Page> <Header /> <StyledHero /> <StyledFaq /> <Footer /> </Page>
```
Comment (connorkrupp) [inline app/pages/blackout/index.jsx]: Did prettier do this?

File: app/pages/blackout/footer.jsx (hunk 1 above)
Comment (roball24) [inline app/pages/blackout/footer.jsx]: will do

File: app/pages/blackout/index.jsx (hunk 2 above)
Comment (roball24) [inline app/pages/blackout/index.jsx]: haha i guess so

**Comment by roball24** [comment]
merging with master now. I'll change those quick

**Pull request merged**